transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/push/NotificationProvider.kt

```diff
@@ -0,0 +1,12 @@
+package com.genesys.cloud.messenger.transport.push
+
+/**
+ * Represents push notification providers for mobile platforms.
+ *
+ * @property APNS Apple Push Notification Service.
+ * @property FCM Firebase Cloud Messaging.
+ */
+enum class NotificationProvider {
```

Comment:
Consider renaming `NotificationProvider` to `PushProvider` since not all push messages result in user-visible notifications

Reply:
Done